src/analysis/special_functions/pow_nnreal.lean

```diff
@@ -0,0 +1,693 @@
+/-
```

Comment:
Similarly, I'm not sure there is much point separating the `real` and `nnreal` stories. Isn't there a more semantic split?